README.md

```diff
@@ -335,7 +356,14 @@ This is an example of usage inside an ACC ViewModel:
 ```
 
 ## Capturing a payment
-ForageSDK will expose the following function to capture a payment:
+
+### Step 1: Add the `ForagePINEditText` UI component
+
+You need to use the `ForagePINEditText` component to your app to check a card's balance. If you need help, refer to the instructions for [adding the UI component](#step-1-add-the-foragepanedittext-ui-component) in the method above. Just change the component name!
+
+### Step 2: Capture the EBT payment
```

Comment:
Can we make this step 3 and add a step 2 which is to call a route on the customer's backend which creates the Payment object. The `ref` of that Payment object is an input to the next step and we also got some feedback from customers that server side interactions are not clear in these docs.